package.json

```diff
@@ -25,17 +38,29 @@
     "posttest": "npm run lint",
     "slug": "ts-node slug.ts",
     "test": "jest",
-    "cow-slug": "node dist/cow-slug.js",
+    "cow-slug": "node lib/cow-slug.js",
```

Comment:
Normally we use à dir called "dist" for autogenerated files. It's better than "lib"

Reply:
haha we removed the dist together and then I saw somewhere this autogenerated lib and I thought, lets call it like that. But if dist is better we go for that.
I changed it back now:)